src/desktop/apps/feature-reaction/client.tsx

```diff
@@ -0,0 +1,28 @@
+import { buildClientApp } from "reaction/Artsy/Router/client"
+import { data as sd } from "sharify"
+import { routes as featureRoutes } from "reaction/Apps/Feature/routes"
+import React from "react"
+import ReactDOM from "react-dom"
+import { loadableReady } from "@loadable/component"
+
+const mediator = require("desktop/lib/mediator.coffee")
+
+buildClientApp({
+  routes: featureRoutes,
+  context: {
+    user: sd.CURRENT_USER,
+    mediator,
+  } as any,
```

Comment:
What's with the casting as any?

Reply:
I admittedly copy-pasted this from @damassi 's viewing room shell work 😬.

Removing the `as any` doesn't appear to cause issues-- @damassi is there some other reason to keep?

Reply:
I think after Alloy left I might have made `SystemContext` less strict (since we frequently use it to pass all kinds of data over from Force), in which case these `any`'s can go away.

He had an idea where we would find a way to create app-level type definitions that extend system context but we haven't had a chance to think about how to implement that, and meanwhile everyone kept using system context in a pretty free form way and adding `any`, so... patterns formed. 

Similar to my [comment here](https://github.com/artsy/reaction/pull/3508#discussion_r422528618) we're getting to a point of stability in the new app architecture where proper types will be valuable. Just have to find the time to do it, while also retrofitting existing code in support.